common/src/main/java/org/apache/uniffle/common/util/RetryUtils.java

```diff
@@ -76,19 +76,24 @@ public static <T> T retryWithCondition(
       Function<Throwable, Boolean> isRetryFunc)
       throws Throwable {
     int retry = 0;
+    Throwable previousThrowable = null;
     while (true) {
       try {
         return cmd.execute();
       } catch (Throwable t) {
         retry++;
         if (isRetryFunc.apply(t) && retry < retryTimes) {
+          previousThrowable = t;
           LOG.info("Retry due to Throwable, " + t.getClass().getName() + " " + t.getMessage());
```

Comment:
This has thrown the exception.

Reply:
Using only `t.getMessage()` will result in the loss of the complete exception stack trace, which will make it difficult to locate issues when they arise.